pkg/ssh/ssh_keys.go

```diff
@@ -14,8 +14,17 @@ import (
 )
 
 type Context struct {
-	ConfigDir string
-	KeygenExe string
+	configDir string
+	keygenExe string
+}
+
+// NewContextForTests creates a new `ssh.Context` with internal properties set to the
+// specified values. It should only be used to testing to inject test-specific setup.
```

Comment:
Minor readability improvement

```suggestion
// specified values. It should only be used to inject test-specific setup.
```

Reply:
I made and pushed this up rather than block merging changes waiting for someone else to do it. 🙇